resources/assets/uad_lists.json

```diff
@@ -14768,6 +14768,15 @@
     "labels": [],
     "removal": "Recommended"
   },
+  {
+    "id": "com.uscc.ecid",
+    "list": "Misc",
+    "description": "Call guardian (anti-spam), possibly the caller ID service.",
```

Comment:
```suggestion
    "description": "Call Guardian (anti-spam), possibly the caller ID service.",
```

Reply:
I'm not really sure. Maybe the callerID or spam filter for U.S. Cellular?

https://en.wikipedia.org/wiki/U.S._Cellular

I don't have US Cellular service to check.